### PR TITLE
Tests/integration/admin user controller

### PR DESCRIPTION
**Pull request by BartoliniAndBorderCollies**
#### Issue
https://trello.com/c/N2pHV7zA

### Description
This branch adds integration tests to cover `AdminUserController` class

### Changes introduced by this pull request
- add no args constructor to `ListUsersForAdminResponseDTO` class
- add all args constructor to `User` class
- add `cascade` to bike field in `User` class
- add integration tests to cover 100% of methods and 100% of lines in `AdminUserController`

### Testing
- [x] all tests are operational and run fine!

### CODE REVIEW

File: src/integration_test/java/com/klodnicki/Bike/v2/rest/controller/AdminUserControllerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    String responseMessage = response.getResponseBody();
+                    User updatedUser = userRepository.findById(user1.getId()).orElseThrow(IllegalArgumentException::new);
+                    assertNotNull(responseMessage);
+                    assertFalse(updatedUser.isAccountValid());
+                    assertEquals("User banned successfully", responseMessage);
+                });
+    }
+
+    @Test
+    public void deleteUser_ShouldDeleteUserFromDatabase_WhenUserIdIsGiven() {
```
Comment (JasinskiSz) [inline src/integration_test/java/com/klodnicki/Bike/v2/rest/controller/AdminUserControllerIntegrationTest.java:90]: This test naming and construction is looking fine.

**Review by JasinskiSz — CHANGES_REQUESTED** [review]
Same issue imo as in the #15 - I'd like the names of those tests changed to something that is really tested inside.

**Comment by BartoliniAndBorderCollies** [comment]
Ok please check now, but this one I haven't changed as for me it clearly describes what it does:
 public void banUser_ShouldReturnResponseEntityAsStringAndSetAccountValidToFalse_WhenUserIdIsGiven()

**Review by JasinskiSz — APPROVED** [review]
That's fine

**Pull request merged**